### PR TITLE
Issue #6

### PR DESCRIPTION
**Pull request by saikeshari**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

### CODE REVIEW

**Comment by saikeshari** [comment]
In this pull request, We have solved issue #6 by changing the source of image and adjusting its height.

File: Gemfile.lock [hunk 1]
```diff
@@ -53,13 +55,22 @@ GEM
...
+    tzinfo (1.2.9)
+      thread_safe (~> 0.1)
+    tzinfo-data (1.2021.5)
+      tzinfo (>= 1.0.0)
     unicode-display_width (1.7.0)
+    wdm (0.1.1)
     webrick (1.7.0)
 
 PLATFORMS
   ruby
```
Comment (grace-omotoso) [inline Gemfile.lock]: Please, remove configuration changes from commit

File: _sass/page.scss [hunk 2]
```diff
@@ -27,6 +27,10 @@ body {
     height: auto;
 }
 
+.banner .logo img {
+    height: 40px;
+}
+
```
Comment (grace-omotoso) [inline _sass/page.scss]: Is this responsive?

**Review by grace-omotoso — CHANGES_REQUESTED** [review]
Remove configuration changes and check image responsiveness

File: Gemfile.lock (hunk 1 above)
Comment (saikeshari) [reply Gemfile.lock]: Yes ma'am.

File: _sass/page.scss (hunk 2 above)
Comment (saikeshari) [reply _sass/page.scss]: I checked it and it was looking fine for all displays, but I will go ahead and make this responsive using em instead of px.

Comment (neha-93) [reply _sass/page.scss]: Made the required changes, removed the configure changes, and made the logo responsive.

**Review by grace-omotoso — APPROVED** [review]

**Pull request merged**